Cargo.toml

```diff
@@ -12,6 +12,7 @@ approx = "0.5.1"
 byteorder = "*"
 chrono = "0.2.25"
 image = "*"
+colorgrad = "*"
```

Comment:
https://github.com/mazznoer/colorgrad-rs

implements colorgradients. Alternative is https://github.com/mazznoer/colorgrad-rs